unified-runtime/source/adapters/cuda/common.cpp

```diff
@@ -129,3 +131,84 @@ void setPluginSpecificMessage(CUresult cu_res) {
   setErrorMessage(message, UR_RESULT_ERROR_ADAPTER_SPECIFIC);
   free(message);
 }
+
+namespace umf {
+
+ur_result_t getProviderNativeError(const char *, int32_t) {
+  // TODO: implement when UMF supports CUDA
+  return UR_RESULT_ERROR_UNKNOWN;
```

Comment:
Shouldn't this be implemented?

Reply:
Probably yes, but in a separate PR. This code has been moved here from [adapters/cuda/usm.cpp](https://github.com/intel/llvm/pull/17411/files#diff-9e41e99c3863ed0fbd9f034da023ab09a0c45dbacc40cb5d86458658dda3810c)